src/Bubble.js

```diff
@@ -161,6 +161,21 @@ const styles = {
       borderTopLeftRadius: 3,
     },
   }),
+  center: StyleSheet.create({
+    container: {
+      flex: 1,
```

Comment:
From `react-native` 0.39, `flex: 1` is not longer necessary. Now, `react-native-gifted-chat` 0.1.0 requieres `react-native` >= 0.40. So, could you please review if this could be removed?